command/genesis/genesis.go

```diff
@@ -198,6 +198,14 @@ func setFlags(cmd *cobra.Command) {
 			"event tracker starting block configuration, which is specified per contract address "+
 				"(format: <contract address>:<start block>)",
 		)
+
+		//Regenesis flag that allows to start from non-empty database
+		cmd.Flags().StringVar(
+			&params.initialStateRoot,
+			"trieroot",
+			"",
+			"path to trie root from corresponding triedb",
```

Comment:
This description isn't entirely correct. It is not path to the trie root, but instead it is trie root itself.

```suggestion
			"trie root from the corresponding triedb",
```